tooling/nargo_cli/tests/stdlib-tests.rs

```diff
@@ -47,15 +47,52 @@ fn run_stdlib_tests() {
     let test_report: Vec<(String, TestStatus)> = test_functions
         .into_iter()
         .map(|(test_name, test_function)| {
-            let status = run_test(
-                &StubbedBlackBoxSolver,
-                &mut context,
-                &test_function,
-                false,
-                None,
-                &CompileOptions::default(),
-            );
-
+            let test_function_has_no_arguments = context
+                .def_interner
+                .function_meta(&test_function.get_id())
+                .function_signature()
+                .0
+                .is_empty();
+
+            let status = if test_function_has_no_arguments {
+                run_test(
+                    &StubbedBlackBoxSolver,
```

Comment:
Can you make a wrapper function at least for this since we do the same check here and in test_cmd.rs? You mentioned we can't have it in run_test but it should at least be possible to do it as a free function somewhere in nargo_cli.